### ISSUE TITLE
test_bls.py test fails with ImportError: dlopen failed :

### ISSUE THREAD

**Issue by shashi278** [astropy/astropy#8582]
I got ImportError: dlopen failed error while playing around with some tests in the freshly installed astropy.
It says:
`ImportError: dlopen failed: cannot locate symbol "wrap_into" referenced by "/data/data/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.7/site-packages/astropy/stats/bls/_impl.cpython-37m.so"`

As you can notice, I'm running python on my Android device with the help of an app called Pydroid3. Now, I don't think this issue would've anything to do with Pydroid3 or Android as I'm able to use GUI libraries and other scientific libraries in there.

I'm a very beginner with astropy and this is my first issue I've ever raised in an open source community so kindly correct me if I missed few details.
Thank you.

**Comment by pllim** [comment]
How exactly where you "playing around with tests"? Version info? Command snippets?

Also, Android app is not something that we officially support.

**Comment by astrofrog** [comment]
There’s no reason we couldn’t support it though, but the question is how to get set up with a developer environment that emulated Android?

**Comment by shashi278** [comment]
@pllim I just tried running the snippet, by traversing down the directory and opening up the code, where I got the error. And just for clarification, I ran other tests as well and all of them are working fine.
Later, following along a tutorial, available on the official website, I figured out that writing `from astropy.visualization import astropy_mpl_style` gives the exact error.

@astrofrog, yes there shouldn't be a reason to not support it. And as I already said, I'm able to work with scientific and ML libraries like numpy, pandas, scipy, sklearn, keras and even tensorflow etc. as well as GUI libraries like Matplotlib, Kivy, PyQt etc. in there, and so I don't think( may be wrong) the problem is in there.

**Comment by astrofrog** [comment]
@shashi278 - could you open up python and run:

import astropy
astropy.test()

And post the full output somewhere (eg gist.github.com) and link to it here?

**Comment by astrofrog** [comment]
Make sure you don’t run this inside the astropy source tree, just run it e.g. from the home directory.

**Comment by shashi278** [comment]
@astrofrog sure here it is: https://gist.github.com/shashi278/4d5181775f1b77394a6c865a5490e632
You can see that many tests are failing but all eventually shows the same error.

**Comment by astrofrog** [comment]
@shashi278 - thanks, and how did you install astropy?

**Comment by shashi278** [comment]
@astrofrog, soon after your last comment, I realized that I didn't use `pip` command to install it rather I installed it through an interface provided in the app to install packages. I tried uninstalling and then reinstalling with `pip --no-cache-dir` option but still no success.
I thought to give it a try with termux(  provides interface for using terminal on android) and boom! It worked. No error!
It seems problem with that app and not with the astropy.
Apologies if the issue bothered you all and big thanks for your times and concern! ☺️
I think, we're now good to close this issue.
